pkg/module/source/source.go

```diff
@@ -0,0 +1,15 @@
+package source
+
+import (
+	ocm "github.com/open-component-model/ocm/pkg/contexts/ocm/compdesc"
+	"github.com/open-component-model/ocm/pkg/contexts/ocm/cpi"
+)
+
+const (
+	ocmIdentityName = "module-sources"
+	ocmVersion      = "v1"
+)
+
+type Source interface {
```

Comment:
If you want to use an interface here, it should probably be defined by the consumer rather than the implementing package.